routes.js

```diff
@@ -5,6 +5,6 @@ const routes = module.exports = nextRoutes();
 // ========================= APP ROUTES =====================
 routes.add('home', '/', 'index');
 routes.add('panel', '/panel', 'panel');
-routes.add('indicator', '/panel/:indicator', 'panel');
+routes.add('indicator', '/panel/:indicator', 'indicator');
 routes.add('compare', '/compare', 'compare');
```

Comment:
I agree with the compare page, I totally forgot about it....

About the indicator URL, I think that we can use /indicator/1,2,3,4